modular_skyrat/modules/customization/modules/mob/living/carbon/human/species/ghoul.dm

```diff
@@ -0,0 +1,282 @@
+/datum/species/ghoul
+	name = "Ghoul"
+	id = "ghoul"
+	limbs_id = "ghoul"
+	say_mod = "gurgles"
+	sexes = FALSE
+	default_color = "c4af7c"
+	species_traits = list(NOEYESPRITES, DYNCOLORS, AGENDER, HAS_FLESH, HAS_BONE)
+	can_have_genitals = FALSE //WHY WOULD YOU WANT TO FUCK ONE OF THESE THINGS?
+	mutant_bodyparts = list("ghoulcolor" = "Tan Necrotic")
+	inherent_traits = list(TRAIT_ADVANCEDTOOLUSER,
+		TRAIT_RADIMMUNE,
+		TRAIT_CAN_STRIP,
+		TRAIT_EASYDISMEMBER,
+	)
+	offset_features = list(OFFSET_UNIFORM = list(0,0), OFFSET_ID = list(0,0), OFFSET_GLOVES = list(0,0), OFFSET_GLASSES = list(0,1), OFFSET_EARS = list(0,1), OFFSET_SHOES = list(0,0), \
+						   OFFSET_S_STORE = list(0,0), OFFSET_FACEMASK = list(0,1), OFFSET_HEAD = list(0,1), OFFSET_FACE = list(0,1), OFFSET_BELT = list(0,0), OFFSET_BACK = list(0,0), \
+						   OFFSET_SUIT = list(0,0), OFFSET_NECK = list(0,1))
+	toxic_food = DAIRY | PINEAPPLE
+	disliked_food = VEGETABLES | FRUIT | CLOTH
+	liked_food = RAW | MEAT
+	payday_modifier = 0.75 //-- "Equality"
+	speedmod = -0.35 //speedy mfer
+	armor = -100 //2x more damage
```

Comment:
This isn't a good, or even useful, way of balancing your species. Consider a maxhp modifier instead